### ISSUE TITLE
help

### ISSUE THREAD

**Issue by ghost** [ballgamer56/VanillaExtended#5]
ok so i am another modification developer and i wonder if you might need help with custom units, i do both spriting and coding for them, could i help you with this mod? (i only want to help because the current unit types available is quite vague).

**Comment by ballgamer56** [comment]
Yes, I do need help with how the unit work on V6 update. Especially how to spawn it and how to make custom spawner to only spawn the unit (like it was before V6)

**Comment by ghost** [comment]
i put the pr for an unit constructor in pull requests feel free to use it if you want

**Comment by The4codeblocks** [comment]
Should this be closed?
